packages/vue2/tests/app/Pages/Links/Data/Object.vue

```diff
@@ -3,13 +3,23 @@
     <span class="text">This is the links page that demonstrates passing data through plain objects</span>
 
     <inertia-link method="GET" href="/dump/get" :data="{ foo: 'get' }" class="get">GET Link</inertia-link>
-    <inertia-link as="button" method="POST" href="/dump/post" :data="{ bar: 'post' }" class="post">POST Link</inertia-link>
+    <inertia-link as="button" method="POST" href="/dump/post" :data="{ bar: 'post' }" class="post"
+      >POST Link</inertia-link
+    >
```

Comment:
Formatting looks weird here; shouldn't it have to be?:

```html
<inertia-link as="button" method="POST" href="/dump/post" :data="{ bar: 'post' }" class="post">
    POST Link
</inertia-link>
```

Reply:
This is just Prettier doing its horrendous work in Vue.